lib/ops/opsservice/shrink.go

```diff
@@ -635,3 +658,25 @@ func (s *site) serfNodeLeave(runner *serverRunner) error {
 	})
 	return trace.Wrap(err)
 }
+
+func (s *site) isTeleportMasterConfigPackageFor(server *ProvisionedServer, loc loc.Locator) bool {
+	configPackage := s.teleportMasterConfigPackage(server)
+	return configPackage.Name == loc.Name && configPackage.Repository == loc.Repository
+}
+
+func (s *site) isTeleportNodeConfigPackageFor(server *ProvisionedServer, loc loc.Locator) bool {
+	configPackage := s.teleportNodeConfigPackage(server)
+	return configPackage.Name == loc.Name && configPackage.Repository == loc.Repository
+}
+
+func (s *site) isPlanetConfigPackageFor(server *ProvisionedServer, loc loc.Locator) bool {
+	// Version omitted on purpose since only repository/name are used for comparison
```

Comment:
nit: comment doesn't indicate why the version is ommitted from comparisons.

second nit: The comment is included on isPlanet... but not isTeleport... even though they appear to be the same?

Reply:
I think it does indicate why the version is not used - since only repository name/package name are relevant.
And teleport packages do not take the version parameter - hence no comment.